### PR TITLE
add estimate to notes

### PR DESCRIPTION
**Pull request by ahujsak**
Asana Task : https://app.asana.com/0/361770107085503/1124421080624366

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: src/actions/CryptoExchangeActions.js [hunk 1]
```diff
@@ -148,7 +148,8 @@ export const shiftCryptoCurrency = () => async (dispatch: Dispatch, getState: Ge
         quoteIdUri,
         payinAddress,
         uniqueIdentifier,
-        supportEmail
+        supportEmail,
+        isEstimate
```
Comment (swansontec) [inline src/actions/CryptoExchangeActions.js]: I don't see where `exchange_notes_metadata_generic2` gets updated to use `%12$s`.

File: src/locales/en_US.js [hunk 2]
```diff
@@ -541,6 +541,8 @@ const strings = {
   modal_addressexplorer_message: 'Show Address in Block Explorer?',
   modal_addressexplorer_null: 'This currency has no explorer site as of now',
   approximately: 'Estimated',
+  fixed_quote: 'Fixed Quote',
+  estimated_quote: 'Fixed Quote',
   estimated_exchange_rate: 'Estimated Exchange Rate',
```
Comment (kylanhurt) [inline src/locales/en_US.js]: @ahujsak This seems like a typo. One should say `Estimated Quote`

Comment (ahujsak) [reply src/locales/en_US.js]: Fixed

File: src/actions/CryptoExchangeActions.js (hunk 1 above)
Comment (ahujsak) [reply src/actions/CryptoExchangeActions.js]: FIxed

**Pull request merged**